precompile/living-assets/src/tests.rs

```diff
@@ -0,0 +1,166 @@
+use super::*;
+use evm::ExitRevert;
+use helpers::*;
+use sp_core::H160;
+use sp_runtime::{DispatchError, DispatchResult};
+
+#[test]
+fn check_selectors() {
+	assert_eq!(Action::CreateCollection as u32, 0x1EAF2516);
+	assert_eq!(Action::OwnerOfCollection as u32, 0xFB34AE53);
+}
+
+#[test]
+fn create_collection_on_mock_succeed_should_succeed() {
+	define_precompile_mock!(Ok(()), Some(H160::zero()));
+
+	let input = "1eaf25160000000000000000000000000000000000000000000000000000000000000000000000000000000000000000b7469c43535c826e29c30d25a9f3a035759cf132";
```

Comment:
Can the code here explain what this input corresponds to? Is it the hex-encoded input to create collection, obtained from elsewhere? Perhaps add a comment line, or rename the input to make its origin explicit.

Reply:
give an explicit name will avoid document it. I agree.

Reply:
Looks like this was not addressed?

Reply:
sgtm